### PR TITLE
Recent chat printed

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#299 
Closes: # (issue number)
#299 
## Proposed changes

Brief description of what is fixed or changed
Recent 17 chats are displayed using 'python3 -m play -wrc' command. The chats are displayed as they are visible in the Whatsapp web such as if some chat has unseen message then the last message id printed and the number of unseen messages is printed as it's visible in the web. I have attached a screenshot which will make more clear.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Recent chat displayed on terminal

Screenshot from 2020-05-30 20-23-43

WhatsApp Image 2020-05-30 at 8 51 36 PM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay/__main__.py [hunk 1]
```diff
@@ -171,6 +178,9 @@ async def get_and_match_args(parser):
 
     elif args.about_changer:
         await about_changer.about_changer()
+
```
Comment (rpotter12) [inline wplay/__main__.py:181]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5612035)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/recent_chats.py  2
- wplay/__main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5612035&bid=18385145)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/recent_chats.py  2
- wplay/__main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5612035&bid=18385145)

**Comment by SinghHimadri** [comment]
@rpotter12 Please review my PR.

**Comment by SinghHimadri** [comment]
@rpotter12 Ihave resolved the conflicts have a look.

**Pull request closed without merging**